docs/reference/how-to/disk-usage.asciidoc

```diff
@@ -136,20 +136,40 @@ PUT index
 // CONSOLE
 
 [float]
+=== Watch your shard size
+
+Larger shards are going to be more efficient at storing data. While it is not possible to give a precise recommendation for how large shards should be, and very large shards come with drawbacks (for example, longer recovery times), it is generally possible to have shard sizes in the 20-30 GB range.
```

Comment:
It's a good question but I don't think so. We are still fighting the "30 GB" heap recommendation, too many people see that number and think it's the magical number where they should set their heap without enough consideration for all the factors involved. Instead, I think that the verbiage is good but we should avoid enshrining specific numbers.